tests/unit/linalg/test_operator.cpp

```diff
@@ -31,23 +31,19 @@ TEST_CASE("Operator", "[Operator]")
    {
       // When cg is in a product (on the right), we require cg->iterative_mode to be false.
 
-      ProductOperator *product = NULL;
-      RAPOperator *rap = NULL;
-      TripleProductOperator *triple = NULL;
```

Comment:
These variables were all lighting up for unused variables (in error) but I opted to fix it anyway.